cli/command/stack/kubernetes/cli.go

```diff
@@ -23,20 +23,28 @@ type Options struct {
 }
 
 // NewOptions returns an Options initialized with command line flags
-func NewOptions(flags *flag.FlagSet) Options {
+func NewOptions(flags *flag.FlagSet, namespace ...string) Options {
```

Comment:
Both `flags` and `namespace` basically come from `cmd.Flags()`; why pass both, if we have all information in `flags`?

Reply:
The way Kubernetes works implies that listing the stacks (or anything else) must be made within the context of a namespace, thus as we want to merge the lists of stacks from different namespaces we create a new ```kubernetes.Options``` for each namespace when iterating over them for ```docker stack ls```.
I'll refactor the code to separate the different behaviors in respect to namespaces.